Gopkg.toml

```diff
@@ -69,6 +69,10 @@
   branch = "master"
   name = "github.com/vrischmann/go-metrics-influxdb"
 
+[[constraint]]
```

Comment:
There should be a `Gopkg.lock` change associated with this... it should happen if you run `dep ensure`. Just add & commit the result.